src/lib/time-formats.js

```diff
@@ -1,14 +1,34 @@
-/* exported formats */
+/* exported getLocalizedDateTime */
 
-export const formats = {
-  default: {
-    short_time: '[@] ha',
-    short_date_time: 'ddd [@] ha',
-    long_date_time: 'ddd MMM D \ [@] ha'
-  },
-  'it': {
-    short_time: '[@] H',
-    short_date_time: 'ddd [@] H',
-    long_date_time: 'ddd D MMM \ [@] H'
+export const getLocalizedDateTime = function (time, format) {
+  const uiLocale = browser.i18n.getUILanguage().replace('_', '-') || 'en-US';
```

Comment:
It didn't work for me (and took a bit to figure out why the add-on wasn't working)
```
(new Date()).toLocaleDateString('zh_TW', {hour: 'numeric'});
/*
Exception: RangeError: invalid language tag: zh_TW
@Scratchpad/3:10:2
*/
````